File: ProjetoEuPolitico/src/data/ApuracaoVotos.lua
ApuracaoVotos = {};

ApuracaoVotos.regiao = nil;
ApuracaoVotos.estado = nil;
ApuracaoVotos.cidade = nil;
ApuracaoVotos.cargo = nil;
ApuracaoVotos.ano = nil;
ApuracaoVotos.nomeCandidato = nil;
ApuracaoVotos.numeroCandidato = nil;
ApuracaoVotos.numeroVotos = nil;
ApuracaoVotos.porcentagemVotos = nil;

function ApuracaoVotos:new(o)
  o = o or {}
  setmetatable (o, self)
  self.__index = self
  return o
end


function ApuracaoVotos:getRegiao()
  return self.regiao;
end
function ApuracaoVotos:setRegiao(regiao)
  self.regiao = regiao;
end

function ApuracaoVotos:getEstado()
  return self.estado;
end
function ApuracaoVotos:setEstado(estado)
  self.estado = estado;
end

function ApuracaoVotos:getCidade()
  return self.cidade;
end
function ApuracaoVotos:setCidade(cidade)
  self.cidade = cidade;
end

function ApuracaoVotos:getCargo()
  return self.cargo;
end
function ApuracaoVotos:setCargo(cargo)
  self.cargo = cargo;
end

function ApuracaoVotos:getAno()
	return self.ano;
end
function ApuracaoVotos:setAno(ano)
	self.ano = ano;
end

function ApuracaoVotos:getNomeCandidato()
	return self.nomeCandidato;
end
function ApuracaoVotos:setNomeCandidato(nomeCandidato)
	self.nomeCandidato = nomeCandidato;
end

function ApuracaoVotos:getNumeroVotos()
	return self.numeroVotos;
end
function ApuracaoVotos:setNumeroVotos(numeroVotos)
	self.numeroVotos = numeroVotos;
end

function ApuracaoVotos:getPorcentagemVotos()
	return self.porcentagemVotos;
end
function ApuracaoVotos:setPorcentagemVotos(porcentagemVotos)
	self.porcentagemVotos = porcentagemVotos;
end

File: ProjetoEuPolitico/Web Services/WSApuracaoVotos/src/br/com/BDApuracaoVotos/ApuracaoVotos.java
package br.com.BDApuracaoVotos;

import javax.jdo.annotations.IdGeneratorStrategy;
import javax.jdo.annotations.PersistenceCapable;
import javax.jdo.annotations.Persistent;
import javax.jdo.annotations.PrimaryKey;

@PersistenceCapable
public class ApuracaoVotos {
	
	@PrimaryKey
	@Persistent(valueStrategy = IdGeneratorStrategy.IDENTITY)
	private String id;
	
	@Persistent
	private String estado;
	
	@Persistent
	private String cidade;
	
	@Persistent
	private String cargo;
	
	@Persistent
	private int numeroCandidato;
	
	@Persistent
	private String candidato;
	
	@Persistent
	private int numeroVotos;
	
	@Persistent
	private double porcentagemVotos;
	
	

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getEstado() {
		return estado;
	}

	public void setEstado(String estado) {
		this.estado = estado;
	}

	public String getCidade() {
		return cidade;
	}

	public void setCidade(String cidade) {
		this.cidade = cidade;
	}

	public String getCargo() {
		return cargo;
	}

	public void setCargo(String cargo) {
		this.cargo = cargo;
	}

	public int getNumeroCandidato() {
		return numeroCandidato;
	}

	public void setNumeroCandidato(int numeroCandidato) {
		this.numeroCandidato = numeroCandidato;
	}

	public String getCandidato() {
		return candidato;
	}

	public void setCandidato(String candidato) {
		this.candidato = candidato;
	}

	public int getNumeroVotos() {
		return numeroVotos;
	}

	public void setNumeroVotos(int numeroVotos) {
		this.numeroVotos = numeroVotos;
	}

	public double getPorcentagemVotos() {
		return porcentagemVotos;
	}

	public void setPorcentagemVotos(double porcentagemVotos) {
		this.porcentagemVotos = porcentagemVotos;
	}
	
	

}
